storage/storage_image.go

```diff
@@ -341,16 +355,31 @@ func (s *storageImageDestination) Commit() error {
 	}
 	logrus.Debugf("created new image ID %q", img.ID)
 	s.ID = img.ID
+	names := []string{}
+	if s.imageRef.DockerReference() != nil {
+		names = append(names, verboseName(s.imageRef.DockerReference()))
+	}
 	if s.Tag != "" {
-		// We have a name to set, so move the name to this image.
-		if err := s.imageRef.transport.store.SetNames(img.ID, []string{s.Tag}); err != nil {
+		// Strip off any digest suffix, and if that leaves anything,
+		// use that name, too.
+		tag := s.Tag
+		if i := strings.Index(tag, "@"); i != -1 {
```

Comment:
No, you're right.  The name that was being attached with an ID suffix was coming from cri-o, so this step didn't need to be taken.